### ISSUE TITLE
Crash report 7.0-rc-1: SQLiteException in SiteSettingsTable

### ISSUE THREAD

**Issue by aforcier** [wordpress-mobile/WordPress-Android#5373]
```
Fatal Exception: android.database.sqlite.SQLiteException: table site_settings has no column named optimizedImage (code 1): , while compiling: INSERT OR REPLACE  INTO site_settings(credsVerified,privacy,postFormats,whitelist,shouldThread,moderationKeys,shouldPage,sendPingbacks,allowComments,categories,threading,relatedPosts,sortBy,id,closeAfter,optimizedImage,defaultPostFormat,address,password,userAccountRequired,shouldCloseAfter,identityRequired,blacklistKeys,tagline,title,defaultCategory,location,username,language,manualApproval,receivePingbacks,paging) VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)
       at android.database.sqlite.SQLiteConnection.nativePrepareStatement(SQLiteConnection.java)
       at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:889)
       at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:500)
       at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:588)
       at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:58)
       at android.database.sqlite.SQLiteStatement.<init>(SQLiteStatement.java:31)
       at android.database.sqlite.SQLiteDatabase.insertWithOnConflict(SQLiteDatabase.java:1471)
       at org.wordpress.android.datasets.SiteSettingsTable.saveSettings(SiteSettingsTable.java:93)
       at org.wordpress.android.ui.prefs.DotComSiteSettings$3.onResponse(DotComSiteSettings.java:152)
       at org.wordpress.android.ui.prefs.DotComSiteSettings$3.onResponse(DotComSiteSettings.java:130)
```

**Comment by maxme** [comment]
Looks like there is a missing case [here](https://github.com/wordpress-mobile/WordPress-Android/blob/9aac40b6100d1374bc02bffd2f469807d01b5230/WordPress/src/main/java/org/wordpress/android/WordPressDB.java#L184-L190)

**Comment by aforcier** [comment]
Fixed in #5378.
